src/mmtk.rs

```diff
@@ -403,35 +401,26 @@ impl<VM: VMBinding> MMTK<VM> {
     }
 
     /// The application code has requested a collection. This is just a GC hint, and
-    /// we may ignore it.
+    /// we may ignore it (See `ignore_system_gc` and `full_heap_system_gc` in [`crate::util::options::Options`]).
```

Comment:
We removed `full_heap_system_gc`.

```suggestion
    /// we may ignore it (See `ignore_system_gc` in [`crate::util::options::Options`]).
```